Features/ManageabilityPkg/Library/IpmiCommandLib/IpmiCommandLibPei.inf

```diff
@@ -0,0 +1,32 @@
+### @file
+# Component description file for IPMI Command Library.
+#
+# Copyright (c) 2024, Ampere Computing LLC. All rights reserved.<BR>
+#
+# SPDX-License-Identifier: BSD-2-Clause-Patent
+#
+###
+
+[Defines]
+  INF_VERSION                    = 0x00010005

```

Comment:
Nit: the INF version is old. I believe  0x0001001E is the latest revision (https://tianocore-docs.github.io/edk2-InfSpecification/draft/#edk-ii-module-information-inf-file-specification).